app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -394,7 +394,9 @@ function fetchNextPageIfNearBottom() {
       fetchNextFollowingPage(indexContainer);
     };
   } else {
-    scrollableElemId = 'articles-list';
+    const isHome = indexWhich === 'home';
+    scrollableElemId = isHome ? 'main-content' : 'articles-list';
```

Comment:
Temporary check until we've refactored the other uses of the 'articles-list' ID to become 'main-content'

Alternatively, I could refactor the other uses of 'articles-list' to 'main-content' in this same PR - it would make it a bit bigger but not massively so.